lib/Doctrine/Common/Cache/PhpFileCache.php

```diff
@@ -108,8 +108,7 @@ private function includeFileForId($id)
     {
         $fileName = $this->getFilename($id);
 
-        // note: error suppression is still faster than `file_exists`, `is_file` and `is_readable`
-        $value = @include $fileName;
+        $value = file_exists($fileName) ? @include $fileName : null;
```

Comment:
It seems that dealing with error_handlers is more expensive than doing a file_exists.
https://3v4l.org/gOXjR
Either way, I'm not sure this kind of micro-optimization is healthy to the code.